common/src/main/java/org/conscrypt/SSLUtils.java

```diff
@@ -253,28 +254,76 @@ static String getClientKeyType(byte clientCertificateType) {
         }
     }
 
+    static String getClientKeyTypeFromSignatureAlg(int signatureAlg) {
```

Comment:
If this is purely meant to get the key type, It might be a better idea to use SSL_get_signature_algorithm_key_type here.

Reply:
Sounds good, done.